README.md

```diff
@@ -0,0 +1,149 @@
+Omigami uses multivariate models to perform a recursive feature elimination within a repeated double cross-validation.
+
+## Install
+
+```sh
+pip install omigami
+```
+
+## Motivation
+
+- It's hard to protect against selection bias on high-demensional omics data ([Krawczuk and Łukaszuk, 2016](https://www.sciencedirect.com/science/article/pii/S0933365715001426)). Even common cross-validation has been shown to overfit.
+- Most feature selection techniques focus on finding the minimal set of strongest features. Omitting redundant variables that are however still relevant to understanding the biochemical systems.
+- There's no freely available and easy-to-use Python tool that implements a minimally biased repeated double cross validation.
+- A robust selection requires many (100 - 5.000) models to be trained. Running such a large number of models in reasonable time, requires non-trivial parallelization.
+
+## Features
+
+- [x] Repeated double cross-validation
+- [x] Multivariate feature selection (Random Forest, XGB or PLS-DA)
+- [x] Minimal optimal and all relevant feature selection
+- [x] Efficient Parallelization (with Dask)
+- [x] Familiar scikit-learn API
+- [ ] Plotting
+- [ ] Predict with trained models
+
+## Usage
+
+### A minimal example
+
+- **test.csv**: This is your omics dataset.
+- **target**: Replace this with the name of the column that denotes your class variable
+  - e.g. this column will contain (1/0, pathological/control, treatment/non-treatment, etc.)
+
+```python
+        import pandas as pd
+        data = pd.read_csv('test.csv')
+        X = data.drop(columns=["target"]).values
+        y = data["target"].values
+```
+
+Once the data is ready, we can get a feature selector, fit it and look at the selected features:
+
+```python
+from omigami.omigami import FeatureSelector
+feature_selector = FeatureSelector(
+    repetitions=10,
+    n_outer=5,
+    n_inner=4
+    estimator="RFC",   # random forest classifier
+    metric="MISS",   # missclassifications
+)
+
+feature_selector.fit(X, y)
+
+selected_features = feature_selector.selected_features
+```
+
+It might take a while for it to complete, depending on your machine and on the model
+selected.
+
+The features are reported as column indexes. To get the names just pass the selection
+to the data frame:
+
+```python
+selected_feature_names = data.columns[list(selected_features["min"])]
+```
+
+### Parallelization
+
+The feature selection can be time consuming. To speed it up execute the various CV loops in parallel using a dask cluster.
+The dask cluster can be remote, or running in local to exploit all the processors of
+the your computer.
+
+For the latter case - which is probably the most common case - it's sufficient to run the following
+**at the beginning of your script**:
+
+```python
+from dask.distributed import Client
+client = Client()
+```
+
+*Also*: Dask gives you a neat dashboard to see the status of all the jobs at `http://localhost:8787/status`.
```

Comment:
we are now keeping it more general, check the new readme. Probably we can move the mention to dask in the notebook example